projects/addon-table/components/table/tr/tr.component.ts

```diff
@@ -48,4 +48,9 @@ export class TuiTrComponent<T extends Record<string, any>> {
         @Inject(forwardRef(() => TuiTbodyComponent))
         private readonly body: TuiTbodyComponent<T>,
     ) {}
+
+    @tuiPure
+    keyof$<U>(key: any): keyof U {
```

Comment:
```suggestion
    $keyof<U>(key: any): keyof U {
```
it is typecasting function (not function which returns Observable).

Reply:
fixed